CHANGELOG.md

```diff
@@ -2,7 +2,8 @@
 
 ## v1.0.0-rc2 (unreleased)
 
-- No changes yet.
+- **[Breaking]** `ValueType` and `GetType` functionality is removed in favor of using
```

Comment:
This is a public API, that no one is using.
It is repeating [reflection package types](https://github.com/uber-go/config/blob/master/value.go#L36) and is [very deceiving](https://github.com/uber-go/config/blob/master/value.go#L64)